### PR TITLE
Update web-application-firewall.mdx

### PR DESCRIPTION
**Pull request by malavikabala**
Fix not supported and limitations

Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 1]
```diff
@@ -45,13 +45,15 @@ You simply enable WAF in Manager UI, and determine the services that you want to
...
 - eBPF dataplane
+- GKE clusters
 - RKE and RKE2 clusters
 
 **Limitations**
 
 - WAF is not supported for host-networked client pods
 - When selecting and deselecting traffic for WAF, active connections may be disrupted
-- You cannot use the annotation on [LoadBalancer type](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) services. 
+- You cannot use the annotation on [LoadBalancer type](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) services.
```
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx:55]: Not sure what "use the annotation" means? Does it mean "you cannot use WAF with LoadBalancer type]() services?
 
- You cannot use the annotation on [LoadBalancer type](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) services.

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 2]
```diff
@@ -45,13 +45,15 @@ You simply enable WAF in Manager UI, and determine the services that you want to
...
+- WAF is incompatible with Egress Gateways
```
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx]:
```suggestion
- WAF is incompatible with egress gateways
```

**Review by bmckercher123 — CHANGES_REQUESTED** [review]
Small fixes. Also, the update is in root for CE, which means 3.18 only.  The rule for generic updates are: current release minus 2. So if applicable backwards, make update should be in: 3.16 (current) to 3.14. Finally, the changes apply to CC docs as well. The root folder for CC is 3.16.

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 1 above)
Comment (malavikabala) [reply calico-enterprise/threat/web-application-firewall.mdx:55]: I'm honestly not sure myself - this was already in the file I edited but it isn't on the docs page for WAF

**Pull request closed without merging**